app/molecules/AdminForm/Editor.tsx

```diff
@@ -15,7 +15,7 @@ export function Editor({ helper, isDisabled = false, label, name, placeholder }:
 
   const hasError = (touched[name] !== undefined || submitCount > 0) && Boolean(errors[name])
   const maybeError = hasError ? String(errors[name]) : undefined
-  const defaultVaLue = useMemo(() => values[name], [])
+  const defaultValue = values[name] || '<br>'
```

Comment:
In fact we should be able to pass an undefined there. I may deserve a TODO for later on.

https://github.com/singularity-ui/core/blob/main/fields/MarkdownEditor/helpers.ts#L19